tests/Doctrine/Tests/Common/Proxy/ProxyMagicMethodsTest.php

```diff
@@ -80,37 +78,36 @@ function (Proxy $proxy, $method, $params) use (&$counter) {
                 $proxy->__setInitializer(null);
 
                 $proxy->publicField = 'modifiedPublicField';
-                $counter            += 1;
+                $counter           += 1;
```

Comment:
Just wondering whether this alignment is correct? `=` are aligned, but `+` is not, it looks strange.

Reply:
AFAIK that's our standard. Maybe we can discuss that on `doctrine/coding-standard`